src/ipc-handlers.ts

```diff
@@ -1084,3 +1085,11 @@ export function addSyncOperation( event: IpcMainInvokeEvent, id: string ) {
 export function clearSyncOperation( event: IpcMainInvokeEvent, id: string ) {
 	ACTIVE_SYNC_OPERATIONS.delete( id );
 }
+
+export function getSiteSize( _event: IpcMainInvokeEvent, siteId: string ) {
+	const site = SiteServer.get( siteId );
+	if ( ! site ) {
+		throw new Error( 'Site not found.' );
+	}
+	return calculateDirectorySize( site.details.path );
```

Comment:
Let's make sure we check the size of wp-content folder.
Something like:

```suggestion
	return calculateDirectorySize( nodePath.join( site.details.path, 'wp-content' ) );
```

Reply:
Thanks, I made these changes in https://github.com/Automattic/studio/pull/811/commits/d3988d71ca82589670eaa83f0cdff5bcb278652a